dandi/models.py

```diff
@@ -113,6 +113,104 @@ def diff_models(model1, model2):
             print(f"{field} is different")
 
 
+def to_datacite(dandiset):
+    from .metadata import migrate2newschema
+
+    meta = dandiset.metadata
+    newmeta = migrate2newschema(meta)
+
+    dandiset_id = dandiset.identifier
+
+    attributes = {}
+    attributes["identifiers"] = [
+        {"identifier": newmeta.doi, "identifierType": "DOI"},
+        # not sure if I can add dandi_id in the PDF documentation the only option is DOI...
+        {"identifier": dandiset_id, "identifierType": "Dandi"},
```

Comment:
indeed they seem to only allow DOIs for identifiers, but you can add the other to `relatedIdentifiers`

Reply:
I might be wrong, but I understood that `relatedIdentifiers` are for other resources that are related to this asset (and not other identifiers for the asset), that's why I added `newmeta.relatedResources` there

Reply:
i will just remove the second identifier for now

Reply:
you can use the relation `IsIdenticalTo`  when adding the DANDI identifier to relatedIdentifiers

Reply:
but in theory I should use a specific list for `relatedIdentifierType`, where there is no "Dandi". I could always try to add and see what happens...

Reply:
you can use URL and put `https://identifiers.org/DANDI:<datasetid>/version`

Reply:
ok